### PR TITLE
simplewallet: remove payment_id and integrated_address 

### PR DESCRIPTION
**Pull request by wowario**

### CODE REVIEW

File: src/simplewallet/simplewallet.cpp [hunk 1]
```diff
@@ -5059,7 +4984,7 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
   // prompt is there is no payment id and confirmation is required
   if (!payment_id_seen && m_wallet->confirm_missing_payment_id() && dsts.size() > num_subaddresses)
   {
-     std::string accepted = input_line(tr("No payment id is included with this transaction. Is this okay?  (Y/Yes/N/No): "));
+     std::string accepted = input_line(tr("No payment id is included with this transaction: ask the recipient to use subaddresses instead. Continue anyway?  (Y/Yes/N/No): "));
```
Comment (SamsungGalaxyPlayer) [inline src/simplewallet/simplewallet.cpp]: Is this a useful string? How about, if sent to a standard address (not a subaddress): `There is no easy way for exchanges and large merchants to identify that this transaction came from you. Request a subaddress in these cases. Continue anyway?`

File: src/simplewallet/simplewallet.cpp [hunk 2]
```diff
@@ -5614,7 +5539,7 @@ bool simple_wallet::sweep_main(uint64_t below, bool locked, const std::vector<st
   // prompt is there is no payment id and confirmation is required
   if (!payment_id_seen && m_wallet->confirm_missing_payment_id() && !info.is_subaddress)
   {
-     std::string accepted = input_line(tr("No payment id is included with this transaction. Is this okay?  (Y/Yes/N/No): "));
+     std::string accepted = input_line(tr("No payment id is included with this transaction: ask the recipient to use subaddresses instead. Continue anyway?  (Y/Yes/N/No): "));
```
Comment (SamsungGalaxyPlayer) [inline src/simplewallet/simplewallet.cpp]: Same comment.

File: src/simplewallet/simplewallet.cpp [hunk 3]
```diff
@@ -5886,7 +5811,7 @@ bool simple_wallet::sweep_single(const std::vector<std::string> &args_)
   // prompt if there is no payment id and confirmation is required
   if (!payment_id_seen && m_wallet->confirm_missing_payment_id() && !info.is_subaddress)
   {
-     std::string accepted = input_line(tr("No payment id is included with this transaction. Is this okay?  (Y/Yes/N/No): "));
+     std::string accepted = input_line(tr("No payment id is included with this transaction: ask the recipient to use subaddresses instead. Continue anyway?  (Y/Yes/N/No): "));
```
Comment (SamsungGalaxyPlayer) [inline src/simplewallet/simplewallet.cpp]: Same comment.

File: src/simplewallet/simplewallet.cpp (hunk 1 above)
Comment (wowario) [reply src/simplewallet/simplewallet.cpp]: Thanks @SamsungGalaxyPlayer for reviewing. I've updated messages as recommended.

**Pull request merged**